ngx_http_auth_ldap_module.c

```diff
@@ -26,6 +26,7 @@
  * SUCH DAMAGE.
  */
 
+#include <sys/socket.h>
```

Comment:
Added `sys/socket.h` for `AF_INET` and `AF_INET6` later in `ngx_http_auth_ldap_ssl_handshake_handler`